packages/utils/src/lib/reports/generate-md-report-categoy-section.ts

```diff
@@ -0,0 +1,148 @@
+import { AuditReport, Table } from '@code-pushup/models';
+import { slugify } from '../formatting';
+import {
+  CATEGORIES_TITLE,
+  reportOverviewTableAlignment,
+  reportOverviewTableHeaders,
+} from './constants';
+import { metaDescription, tableSection } from './formatting';
+import { style as htmlFontStyle } from './html/font-style';
+import {
+  SPACE,
+  h2,
+  h3,
+  indentation,
+  li,
+  link,
+  paragraphs,
+  section,
+  style,
+} from './md';
+import { ScoredGroup, ScoredReport } from './types';
+import {
+  countCategoryAudits,
+  formatReportScore,
+  getPluginNameFromSlug,
+  getSortableAuditByRef,
+  getSortableGroupByRef,
+  scoreMarker,
+} from './utils';
+
+export function categoriesOverviewSection(
+  report: Pick<ScoredReport, 'categories' | 'plugins'>,
+): string {
+  const { categories, plugins } = report;
+  if (categories.length > 0 && plugins.length > 0) {
+    const tableContent: Table = {
+      headings: reportOverviewTableHeaders,
+      rows: categories.map(({ title, refs, score }) => ({
+        // @TODO shouldn't this be the category slug as title is not unique? => slugify(title) -> slug
+        category: link(`#${slugify(title)}`, title),
```

Comment:
The heading "ID" is inferred from the heading text in Markdown. Although in practice our `slug`s usually match the slugified `title`, this is not enforced anywhere, so might not work as an anchor link.